migaloo/assetlist.json

```diff
@@ -865,6 +865,58 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/syn.png"
       }
+    },
+    {
+      "description": "Dragon Token is the native staking and governance token of Dungeon Chain, serving as the backbone of the ecosystem. Its primary roles include staking, where users stake Dragon Tokens to secure the network, enhance its functionality, and earn rewards in return; governance, granting token holders voting rights to influence key decisions such as game onboarding, protocol updates, and community-driven initiatives; and ecosystem growth, facilitating transactions, incentivizing developers, and promoting the creation of new interchain games. Dragon Token ensures a decentralized, fair, and community-driven ecosystem, fostering the evolution of blockchain-based gaming. It plays a crucial role in driving the growth of Dungeon Chain by empowering both developers and players to participate actively in the ecosystem's governance and success.",
+      "denom_units": [
+        {
+          "denom": "ibc/D8C6D71032EC7CC4C9CCD0FF709CF87D34A7311FE5491E70F7EDE2351D189C10",
+          "exponent": 0
+        },
+        {
+          "denom": "DGN",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/D8C6D71032EC7CC4C9CCD0FF709CF87D34A7311FE5491E70F7EDE2351D189C10",
+      "name": "Dragon",
+      "display": "DGN",
+      "symbol": "DGN",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "dungeon",
+            "base_denom": "udgn",
+            "channel_id": "channel-6"
+          },
+          "chain": {
+            "channel_id": "channel-142",
+            "path": "transfer/channel-142/uluna"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dungeon/images/DGN.png"
+      },
+      "coingecko_id": "dragon-coin-2",
```

Comment:
Please remove the coingecko id here. It's already been registered to the original Dragon coin in Dungeon Chain.

Reply:
I'm new to the world of Github and PRs. How do I make this change without creating a whole other PR?

Reply:
I changed this in my patch-1 fork and committed it